ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -6,25 +6,25 @@ namespace ConwaysGameOfLife.Tests
 {
     public static class UniverseHelper
     {
-        public static bool UniversesAreEqual(Universe obj1, Universe obj2)
+        public static bool UniversesAreEqual(Universe universe1, Universe universe2)
         {
-            var isSizeSame = (obj1.GridWidth == obj2.GridWidth) && (obj1.GridLength == obj2.GridLength);
+            var isSizeSame = (universe1.GridWidth == universe2.GridWidth) && (universe1.GridLength == universe2.GridLength);

```

Comment:
Extra consideration: this is actually only checking the grid dimensions - what about if the actual number of `Cells` were to be different? Could be an extra check early on which is just as simple as this existing one